src/main/java/se/sundsvall/smloader/integration/openemapper/orderingrecruitmentsupport/OrderingRecruitmentSupport.java

```diff
@@ -0,0 +1,61 @@
+package se.sundsvall.smloader.integration.openemapper.orderingrecruitmentsupport;
+
+import java.util.List;
+import se.sundsvall.smloader.integration.util.annotation.GenericType;
+import se.sundsvall.smloader.integration.util.annotation.XPath;
+
+record OrderingRecruitmentSupport(
+
+	@XPath("/FlowInstance/Header/Flow/FamilyID") String familyId,
+	@XPath("/FlowInstance/Header/FlowInstanceID") String flowInstanceId,
+	@XPath("/FlowInstance/Header/Status/Name") String status,
+
+	@XPath("/FlowInstance/Values/applicant/firstname") String applicantFirstname,
+	@XPath("/FlowInstance/Values/applicant/lastname") String applicantLastname,
+	@XPath("/FlowInstance/Values/applicant/username") String applicantUserId,
+	@XPath("/FlowInstance/Values/applicant/email") String applicantEmail,
+	@XPath("/FlowInstance/Values/applicant/phone") String applicantPhone,
+	@XPath("/FlowInstance/Values/applicant/title") String applicantTitle,
+	@XPath("/FlowInstance/Values/applicant/organization") String applicantOrganization,
+
+	@XPath("/FlowInstance/Values/recrutingManager/firstname") String recruitingManagerFirstname,
```

Comment:
recrutingManager is spelled wrong. Intentional?

Reply:
Yes, the tag from oep is "<recrutingManager>"